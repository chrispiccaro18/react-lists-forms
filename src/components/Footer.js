import React from 'react';
import PropTypes from 'prop-types';

function Footer({ title }) {
  return (
    <footer>
      <h2>{title}</h2>
    </footer>
  );
}

Footer.propTypes = {
  title: PropTypes.string.isRequired
};

export default Footer;
