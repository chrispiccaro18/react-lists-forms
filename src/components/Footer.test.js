import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer component', () => {
  it('renders a Footer', () => {
    const wrapper = shallow(<Footer title="Lauren" />);
    expect(wrapper).toMatchSnapshot();
  });
});
