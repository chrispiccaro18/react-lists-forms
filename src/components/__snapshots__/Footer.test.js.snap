// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a Footer 1`] = `
<footer>
  <h2>
    Lauren
  </h2>
</footer>
`;
